lib/rules/forbid-prop-types.js

```diff
@@ -170,6 +172,7 @@ module.exports = {
       CallExpression(node) {
         if (
           node.arguments.length > 0
+          && `${pragma}.PropTypes`
```

Comment:
should this be comparing this string to anything? otherwise it's just a noop and could be deleted.